app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js

```diff
@@ -45,6 +45,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var shouldHaveCookieMessage = (this.$module && window.GOVUK.cookie('cookies_preferences_set') !== 'true')
 
       if (shouldHaveCookieMessage) {
+        this.$module.removeAttribute('hidden')
```

Comment:
If we're hiding the cookie banner unless JS is enabled, I think there's some additional code that we can change or possibly remove. In the [setupCookieMessage function](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js#L19) there's some code that makes the two buttons appear, because they're hidden by default so they're not shown when JS is disabled.

There's also some stuff in the [showCookieMessage function](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js#L57) where its setting the display to 'none', which also might not be needed anymore? I think it's making the cookie banner hidden on the cookies settings page if you've not made a choice yet, but if it's hidden by default now that might not be needed.

Reply:
Thanks, should be fixed now 👍